src/hardware/serialport/softmodem.h

```diff
@@ -20,10 +20,12 @@
 #ifndef DOSBOX_SERIALMODEM_H
 #define DOSBOX_SERIALMODEM_H
 
+#include <vector> // vector
+#include <memory> // unique_ptr
```

Comment:
I think we don't need a comment after include :)

Reply:
Will clean up.

Reply:
Fixed in f5319290